src/lib/draw/Toolbox.svelte

```diff
@@ -46,6 +49,35 @@
   let splitRouteMode: SplitRouteMode;
   let streetViewMode: StreetViewMode;
 
+  const eventHandlers: { [mode in Mode]: EventHandler } = {
+    "edit-attribute": new DefaultEventHandler(),
+    "edit-geometry": new DefaultEventHandler(),
+    route: new DefaultEventHandler(),
+    point: new DefaultEventHandler(),
+    "free-polygon": new DefaultEventHandler(),
+    "snap-polygon": new DefaultEventHandler(),
+    "split-route": new DefaultEventHandler(),
+    "street-view": new DefaultEventHandler(),
+  };
+
+  MapEvents.forEach((eventName) => {
```

Comment:
To let hot-module reloading work, we need to remove all these handlers in `onDestroy`. I _think_ to do that, we have to be careful about passing in exactly the same closure here when we do `map.off`. Not sure yet about the nicest way to do that

Reply:
Ahhhh good point, I did at some point think I needed to do map.off, didn't consider the closure and forgot to do it thought 😅